Sources/VitalDevices/DeviceReading/GATTMeter.swift

```diff
@@ -87,10 +88,13 @@ internal class GATTMeter<Sample> {
   }
 
   func pair(device: ScannedDevice) -> AnyPublisher<Void, Error> {
-    _pair(device: device).map { _ in ()}.eraseToAnyPublisher()
+    _connect(device: device, then: { _, _ in Just(()).setFailureType(to: Error.self) })
   }
```

Comment:
why the private name change from `_pair` to `_connect`?

Reply:
So let's change line 90 to be called connect?

Reply:
Line 90 is the user-facing `pair()` function, which is used to force pairing but is a no-op if already paired else. The parity exists in Android as well, which has the concept of explicit pairing/bonding.

It relies on the fact that Core Bluetooth will auto-start pairing when one attempts to use BLE characteristics that require bonding and/or encryption.

Reply:
(and there are legitimate UX e.g. onboarding/first-time setup where you want to try to explicitly pair)